### ISSUE TITLE
Emails from Right to Know aren't being delivered due to no DKIM / DMARC

### ISSUE THREAD

**Issue by benrfairless** [openaustralia/infrastructure#167]
Emails to specific authorities aren't being sent to some authorities. See RTK issue https://github.com/openaustralia/righttoknow/issues/136

According to the user who is reporting the issue, this is likely caused because we aren't using DKIM or DMARC - both of which are now accepted as industry standards for sending email.

We should work on enabling DKIM first, then when we know that DKIM is working we should update DMARC.

**Comment by benrfairless** [comment]
Missive conversation: https://mail.missiveapp.com/#inbox/conversations/cc6084ea-40c7-4c2d-9de8-5d7f2a191cb5

**Comment by benrfairless** [comment]
Missive conversation: https://mail.missiveapp.com/#inbox/conversations/9972d15f-feba-4ebb-96ac-1323d4f66143

**Comment by benrfairless** [comment]
Missive conversation: https://mail.missiveapp.com/#inbox/conversations/22ca7d12-a989-4c16-857e-b6cabd67785a

**Comment by benrfairless** [comment]
Some possible solutions to this problem:

1. Work on enabling DKIM via Google Apps (where emails are currently sent)
2. Use Cuttlefish instead of Google Apps for emails from Right to Know

**Comment by benrfairless** [comment]
@katska is this something you can review urgently for me please?

**Comment by mlandauer** [comment]
Incoming emails to righttoknow come via google apps but outgoing emails are sent directly by postfix on the righttoknow server. Also unfortunately we can't send the emails to cuttlefish because that would break all the delivery information shown on righttoknow.org.au.

Instead we have to set up dkim with postfix which is not a whole lot of fun.

This is fixed by b3079fb1043d8cd10fb0bc8aca13221a157439de...d104f6ec379350de9eede2132bb2a0867a76a173

I did a test of sending email from the server by running:
```
echo "Test postfix and opendkim" | mail -s "Postfix test" -r contact@righttoknow.org.au <my email address>
```
This was sent to my personal gmail account and gmail showed it as passing DKIM and DMARC (and SPF). So, it looks good.

**Comment by mlandauer** [comment]
Note to self: Check out the DMARC reports in a week or so and make sure things look good there too.
